specs/chains/chains.spec

```diff
@@ -0,0 +1,41 @@
+PIPELINES-27
+# Tekton Chains tests
+
+Precondition:
+* Validate Operator should be installed
+* Create signing-secrets for tekton chains
+
+## Using Tekton Chains to create and verify task run signatures: PIPELINES-27-TC01
+Tags: chains, e2e, taskrun
```

Comment:
@VeereshAradhya should we add `sanity` tag here? The issue is the second precondition, users might not have Quay repo creds configured but that will not be a case in our CI, WDYT?

Reply:
:heavy_plus_sign: